sema/commons/j2/j2_functions.py

```
@@ -120,6 +129,12 @@ def xsd_format_string(content, quote, suffix):
     "xsd:datetime": xsd_format_datetime,
     "xsd:anyuri": xsd_format_uri,
     "xsd:string": xsd_format_string,
+    "xsd:gyear": xsd_format_gyear,
+    "year": xsd_format_gyear,
+    "yyyy": xsd_format_gyear,
+    "xsd:gmonthyear":  xsd_format_gmonthyear,
+    "month-year":  xsd_format_gmonthyear,
+    "yyyy-mm":  xsd_format_gmonthyear
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)